src/Kovan-DssSpell.sol

```diff
@@ -16,39 +16,284 @@
+    function init(bytes32) external;
+}
+
+interface Hopeable {
+    function hope(address) external;
+}
+
+interface Kissable {
+    function kiss(address) external;
+}
+
+interface RwaLiquidationLike {
+    function ilks(bytes32) external returns (string memory,address,uint48,uint48);
+    function init(bytes32, uint256, string calldata, uint48) external;
+}
+
+interface Bumpable {
+    function bump(bytes32, uint256) external;
+}
 
 contract DssSpellAction is DssAction {
 
-    string public constant description = "Kovan Spell";
+    // Provides a descriptive tag for bot consumption
+    // This should be modified weekly to provide a summary of the actions
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/2a7a8c915695b7298fe725ee3dc6c613fa9d9bbe/governance/votes/Executive%20vote%20-%20April%2012%2C%202021.md -q -O - 2>/dev/null)"
+    string public constant description = "RWA003-RWA006 integration";
 
-    // Turn off office hours
-    function officeHours() public override returns (bool) {
-        return false;
-    }
+    // Many of the settings that change weekly rely on the rate accumulator
+    // described at https://docs.makerdao.com/smart-contract-modules/rates-module
+    // To check this yourself, use the following rate calculation (example 8%):
+    //
+    // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
+    //
+    // A table of rates can be found at
+    //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
+    //
+    uint256 constant TWO_PCT            = 1000000000627937192491029810;
+    uint256 constant FOUR_PT_FIVE_PCT   = 1000000001395766281313196627;
+    uint256 constant SIX_PCT            = 1000000001847694957439350562;
+    uint256 constant SEVEN_PCT          = 1000000002145441671308778766;
 
-    uint256 constant WAD = 10**18;
-    uint256 constant RAY = 10**27;
-    uint256 constant RAD = 10**45;
+    // Math
+    uint256 constant THOUSAND = 10 ** 3;
+    uint256 constant MILLION  = 10 ** 6;
+    uint256 constant WAD      = 10 ** 18;
+    uint256 constant RAY      = 10 ** 27;
+    uint256 constant RAD      = 10 ** 45;
 
-    address constant MCD_FLASH = 0x5aA1323f61D679E52a90120DFDA2ed1A76E4475A;
+    // Maker changelog 
+    address public constant MAKER_CHANGELOG = 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F;
 
     function actions() public override {
-        // ---------------------------- Add MCD_FLASH ----------------------------
-        address MCD_VAT = DssExecLib.vat();
-        Fileable(MCD_FLASH).file("max", 500_000_000 * WAD);
-        Fileable(MCD_FLASH).file("toll", 5 * WAD / 10000);
-        DssExecLib.authorize(MCD_VAT, MCD_FLASH);
-        DssExecLib.setChangelogAddress("MCD_FLASH", MCD_FLASH);
-
-        // ---------------------------- Update Chainlog version ----------------------------
-        DssExecLib.setChangelogVersion("1.9.1");
+        CentrifugeCollateralValues memory RWA003 = CentrifugeCollateralValues({
```

Comment:
Can we make adding a link to the corresponding governance poll in a comment a convention? E.g. for this one we'd link to:

https://vote.makerdao.com/polling/QmRCn7Mh#poll-detail

The other three are:
https://vote.makerdao.com/polling/QmRWg6nZ#poll-detail
https://vote.makerdao.com/polling/QmZCPDVY#poll-detail
https://vote.makerdao.com/polling/QmdHV2tW#poll-detail

Reply:
Updated, thanks for the links!